packages/forms/frontend/sirius-components-widget-reference/src/ReferencePropertySection.tsx

```diff
@@ -34,14 +38,22 @@ import {
   GQLEditReferenceVariables,
   GQLErrorPayload,
   GQLReferenceWidget,
+  GQLReferenceWidgetStyle,
   GQLSuccessPayload,
 } from './ReferenceWidgetFragment.types';
 
-const useStyles = makeStyles((_) => ({
+const useStyles = makeStyles<Theme, GQLReferenceWidgetStyle>(() => ({
   root: {
     overflow: 'auto',
     maxHeight: 300,
   },
+  style: {
+    color: ({ color }) => (color ? color : 'inherit'),
```

Comment:
undefined instead of 'inherit'
(apply for all properties here)

Reply:
I changed it to null